DigitalLearningSolutions.Web/Helpers/AccountDetailsDataHelper.cs

```diff
@@ -53,7 +53,11 @@ int centreId
                 formData.FirstName!,
                 formData.LastName!,
                 formData.Email!,
-                formData.AliasId
+                formData.AliasId,
+                formData.HasProfessionalRegistrationNumber == true
+                    ? formData.ProfessionalRegistrationNumber
+                    : null,
+                formData.HasProfessionalRegistrationNumber.HasValue

```

Comment:
Does this accurately represent the HasBeenPromptedForPrn? As this is the admin editing the delegate, if the delegate had not been prompted for PRN, but the admin put one in, I'd argue that the user still has not been prompted for the PRN. But then I am not too familiar with this work and what that is used for. Are we good with setting this as true in all cases where the admin supplies it, or false when the admin overrides it with empty?

Reply:
That's a good question, this is the same code that we're using on the my account edit. @SteveJacksonSoft Do we want this as true when the admin has gone through this page, or only on the my account/registration?

Reply:
Hmm, maybe bad naming. `HasBeenPromptedForPrn` is meant to mean "someone's answered the PRN question".

Reply:
So it should be set to true.
It's there so we can distinguish between the two cases which result in an empty PRN field:
1. The user has told us they have no PRN
2. The user already existed when we added the PRN field and we haven't been told whether they have one yet

Reply:
I can't really think of anything much better, so have just left a comment on the DelegateUser property.